starter/orchestrations/CounterOrchestration.cs

```diff
@@ -0,0 +1,27 @@
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+
+namespace Spike.Inventory
```

Comment:
same naming comments here

Reply:
Point of time, will update

Reply:
Renamed Class, Function attribute and Namespace. Drops the word Spike from all places including project name.

New namespace - `InventoryManagement`